app/controllers/provider/admin/registry/policies_controller.rb

```diff
@@ -4,9 +4,9 @@ class Provider::Admin::Registry::PoliciesController < Provider::Admin::BaseContr
   activate_menu :account, :integrate, :policies
   EXAMPLE_SCHEMA = {
       '$schema': 'http://apicast.io/policy-v1/schema#manifest#',
-      name: '[Name of the policy]',
-      summary: '[A brief description of what it does.]',
-      version: '[0.0.1]',
+      name: 'Name of the policy',
```

Comment:
I think it is good to leave those brackets. As we validate those fields, if the user does not edit them they will be invalid and the policy cannot be saved

Reply:
the apicast manifest fails to validate the version, the regex doesn't expects those brackets.

Reply:
Yes that is what I am saying, so the customer will not save this by inadvertence.
Or if you can make them as placeholder and not real values?

Reply:
Thing is the form will fail to render by default. This way it will be shown and communicating the user the right way to do it.